src/processing/script/header.js

```diff
@@ -6,14 +6,18 @@
 import reEscape from '../../utils/regexp-escape';
 import INTERNAL_PROPS from '../../processing/dom/internal-properties';
 import INSTRUCTION from './instruction';
-
+import { USE_STRICT_DIRECTIVE } from './strict';
 
 export const SCRIPT_PROCESSING_START_COMMENT      = '/*hammerhead|script|start*/';
 export const SCRIPT_PROCESSING_END_COMMENT        = '/*hammerhead|script|end*/';
 export const SCRIPT_PROCESSING_END_HEADER_COMMENT = '/*hammerhead|script|processing-header-end*/';
 
+const USE_STRICT         = '"' + USE_STRICT_DIRECTIVE + '";';
+const STRICT_PLACEHOLDER = '{strict-placeholder}';
+
 const HEADER = [
     SCRIPT_PROCESSING_START_COMMENT,
+    STRICT_PLACEHOLDER,
```

Comment:
To be able to insert the directive into our header. 

See https://github.com/DevExpress/testcafe-hammerhead/pull/844#discussion_r83164599